### PR TITLE
Final Pull Request

### PR DESCRIPTION
**Pull request by prakharrathi25**
### Why This PR Adds Value

This is our final pull request for the week close. I have avoided the description since the website is hosted on https://gitdash.tech/ 

Cheers to the team @anandrajaram21 @Green-Ranger11 and myself!

### CODE REVIEW

File: .github/workflows/api-build-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          echo "$ssh_key" > ~/ssh_key
+          chmod 600 ~/ssh_key
+        env:
+          ssh_key: ${{ secrets.CICD_SSH }}
+      - name: Push Files
+        run:    
+          rsync -avrzP ./API/bin/Release/net5.0/publish/ --no-perms -O -e "ssh -i ~/ssh_key -o StrictHostKeyChecking=no" "$user"@"$target_ip":/var/gitdash/api
+        env:
+          user: ${{ secrets.CICD_USER }}
+          target_ip: ${{ secrets.LINODE_IP }}
```
Comment (robzwolf) [inline .github/workflows/api-build-deploy.yaml:40]: I love GitHub Actions! Awesome to see you using this.

File: .gitignore [hunk 2]
```diff
@@ -117,4 +117,7 @@ dist
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+# Useless macOS files
```
Comment (robzwolf) [inline .gitignore:122]: I feel like saying `useless` is a _bit_ harsh 😆 

https://en.wikipedia.org/wiki/.DS_Store

File: README.md [hunk 3]
```diff
@@ -62,6 +96,24 @@ You can start editing the page by modifying `pages/index.tsx`. The page auto-upd
...
+```
+Where:
+ - `{host}` is your host name 
+ - `{port}` is your port number
+ - `{user}` is your user to connect to cockroachDB 
+ - `{password}` is your password to connect to cockroachDB 
+ - `{cluster}` is your cluster name 
+ - `{certificate location}` is the directory location to your certificate (Cloud cockroachDB requires SSL)
+
+
```
Comment (robzwolf) [inline README.md:116]: This README is great 👌

File: server/root.crt [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+yBqJ3j8iUwe5Yt/qfSrRf1/D5R58duaJ+IvLRXeASRqEL+VkDXrW3qOCAQgwggEE
+MA4GA1UdDwEB/wQEAwIBhjAdBgNVHSUEFjAUBggrBgEFBQcDAgYIKwYBBQUHAwEw
+EgYDVR0TAQH/BAgwBgEB/wIBADAdBgNVHQ4EFgQUbZkq9U0C6+MRwWC6km+NPS7x
+6kQwHwYDVR0jBBgwFoAUfEKWrt5LSDv6kviejM9ti6lyN5UwMgYIKwYBBQUHAQEE
+JjAkMCIGCCsGAQUFBzAChhZodHRwOi8veDIuaS5sZW5jci5vcmcvMCcGA1UdHwQg
+MB4wHKAaoBiGFmh0dHA6Ly94Mi5jLmxlbmNyLm9yZy8wIgYDVR0gBBswGTAIBgZn
+gQwBAgEwDQYLKwYBBAGC3xMBAQEwCgYIKoZIzj0EAwMDaAAwZQIxAPJCN9qpyDmZ
+tX8K3m8UYQvK51BrXclM6WfrdeZlUBKyhTXUmFAtJw4X6A0x9mQFPAIwJa/No+KQ
+UAM1u34E36neL/Zba7ombkIOchSgx1iVxzqtFWGddgoG+tppRPWhuhhn
+-----END CERTIFICATE-----
```
Comment (robzwolf) [inline server/root.crt:139]: Should this really be committed to GitHub?

**Review by robzwolf — APPROVED** [review]
Awesome work getting this one across the line, folks! You've done really well to finish this up and I love the detail in your README.

File: server/root.crt (hunk 4 above)
Comment (Green-Ranger11) [reply server/root.crt:139]: Yes Robbie you're absolutely correct. I should've shared this privately to the others as cockroachDb needs an SSL for connecting to it and I'm not sure how to generate more certs for cockroach, it only gives me one :/

**Pull request merged**